src/hooks/usePreloadImg.tsx

```diff
@@ -0,0 +1,14 @@
+import { useEffect, useState } from 'react'
+
+export const usePreloadImg = (thumbnailUrl: string | undefined) => {
```

Comment:
Why do we need this? AssetsManager already does that via `testAssetDownload`

Reply:
wanted to preload the image somehow should we use `testAssetDownload` instead?

Reply:
but the image is already loaded by the assets manager

Reply:
then I'm not sure what is to be done here exactly